sortfilterproxymodel.qbs

```diff
@@ -1,12 +1,14 @@
 import qbs 1.0
+import qbs.File
+import qbs.FileInfo
 
 Product
 {
-    name: "sortfilterproxymodel"
-    type: "dynamiclibrary"
+    name: "SortFilterProxyModel"
+    type: ["dynamiclibrary", "copied qbs resources"]
```

Comment:
@johnbivens btw this really is a hack put in place for now, as we are using

destinationDirectory: project.deployDirectory + "/" + name

instead of actual install / deploy features provided by QBS. This should be cleaned up along with the rest once we start that cleanup.